src/services/config.js

```diff
@@ -27,3 +27,8 @@ if (admin_pass) {
     pass: admin_pass
   }
 }
+
+config.auth_enabled = {
```

Comment:
Agree with Chris - `config.auth.basic_enabled` matches our (not super-strict, but let's make an effort) config naming scheme better: `config.[module].[option]`